testplan/testing/multitest/entries/stdout/assertions.py

```diff
@@ -463,10 +463,22 @@ def get_assertion_details(self, entry):
 
 @registry.bind(assertions.Fail)
 class FailRenderer(AssertionRenderer):
-    def get_header_text(self, entry):
-        """Always return text in red as it is an explicit failure."""
-        return Color.red(entry.description)
+    def get_header(self, entry):
+        if entry.description:
+            return Color.red(entry.description)
+        elif isinstance(entry.message, str):
+            return Color.red(str(entry.message))
```

Comment:
Not sure if we need this.

Reply:
agree, i think we always have a description